pandas/tests/indexing/test_loc.py

```diff
@@ -2700,3 +2700,13 @@ def test_loc_setitem(self, string_series):
         string_series.loc[d2] = 6
         assert string_series[d1] == 4
         assert string_series[d2] == 6
+
+    def test_loc_dict_assign(self):
```

Comment:
Coudl you do something like test_loc_assign_dict_to_row. Maybe test would be better located in tests/frame/indexing but @jbrockmendel is the expert here

Reply:
Sure thing. Let me know if it needs moving and I'll follow through. Still pretty knew to Pandas' testing structure :sweat_smile: